core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -735,7 +735,8 @@ private void internalStart(boolean block) {
             return;
          }
 
-         log.debugf("Starting cache manager %s", configurationManager.getGlobalConfiguration().transport().nodeName());
+         log.debugf("Starting cache manager %s on %s", globalConfiguration.cacheManagerName(),
+                    globalConfiguration.transport().nodeName());
```

Comment:
probably removed the `nodeName()` from the log entry. Not sure if it is useful (or even set!)

Reply:
It is definitely set in our test suite, see `TestCacheManagerFactory#setNodeName`.